src/main/kotlin/coverosR3z/timerecording/TimeRecordingUtilities.kt

```diff
@@ -10,6 +11,11 @@ import coverosR3z.persistence.EmployeeIntegrityViolationException
 class TimeRecordingUtilities(private val persistence: ITimeEntryPersistence) {
 
     fun recordTime(entry: TimeEntryPreDatabase): RecordTimeResult {
+        val user = CurrentUser.get()
```

Comment:
could also add an invariant here - by golly, we better find a current user here.